src/util/numeric.cpp

```diff
@@ -46,11 +46,22 @@ void myrand_bytes(void *out, size_t len)
 	g_pcgrand.bytes(out, len);
 }
 
+float myrand_float()
+{
+	u32 uv = g_pcgrand.next();
+	return (float)uv / (float)((u32)-1);
```

Comment:
less confusing this way:
```suggestion
	return (float)uv / (float)U32_MAX;
```